CancelBooking UseCase/CancelBooking.java

```diff
@@ -4,6 +4,7 @@
 import java.time.LocalDate;
 // Data Layer
 // Represents a reservation with details like customer name, room type, room number, price, CPR, phone number, and booking date.
+
```

Comment:
```suggestion
you can make the getters more secure by marking the fields as (final) and this ensures that once a reservation is created, it can’t be modified. Also you have to ensure input values are validated before storing them. The (toStoring) method may reveal too much sensitive information like CPR, you can create more secure version or remove sensitive details.
```